scripts/run_cpp_linters.py

```diff
@@ -32,35 +33,46 @@ def get_list_of_cpp_files(root_folder: os.PathLike) -> list[str]:
     return ret
 
 
-def run_clang_format(root_folder: os.PathLike):
+def run_clang_format(root_folder: os.PathLike, file_list_str: str) -> None:
```

Comment:
Generally you wouldn't mention the type in the variable name, e.g.
```suggestion
def run_clang_format(root_folder: os.PathLike, files: str) -> None:
```

Up to you to decide what you think is best here.

Reply:
Up to you 😄